### PR TITLE
Login views

### PR DESCRIPTION
**Pull request by AaronRDurant**
As per Exercise 3.4. Currently, the project won't build properly and I'm not sure why, so I need a little help.

### CODE REVIEW

File: src/components/main-view/main-view.jsx [hunk 1]
```diff
@@ -1,48 +1,126 @@
 import React from 'react';
 import axios from 'axios';
 
-import { MovieCard } from '../movie-card/movie-card';
 import { MovieView } from '../movie-view/movie-view';
+import { MovieCard } from '../movie-card/movie-card';
+import { LoginView } from '../login-view/login-view';
+import { RegistrationView } from '../registration-view/registration-view';
```
Comment (eadsimone) [inline src/components/main-view/main-view.jsx:7]: here is the import but you use other name.. you should define as 
import { RegisterView as  RegistrationView } from '../registration-view/registration-view';
or put the right name

File: src/components/main-view/main-view.jsx [hunk 2]
```diff
@@ -1,48 +1,126 @@
...
+										key={movie._id}
+										movie={movie}
+										onClick={(movie) => this.onMovieClick(movie)}
+										/>
+									</Col>
+								))}
+							</Row>
+						</Container>
+					)}
+				</div>
```
Comment (eadsimone) [inline src/components/main-view/main-view.jsx:123]: you forgot to add one more div should be 
```
<React.Fragment>
				<div className='main-view'>
				...
</div>
			</React.Fragment>
```

File: src/components/login-view/login-view.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+						placeholder='Your username here!'
+					/>
+				</Form.Group>
+				<Form.Group controlId='formBasicPassword'>
+					<Form.Label>Password</Form.Label>
+					<Form.Control
+					type='password'
+					value={password}
+					onChange={(e) => setPassword(e.target.value)}
+					type='password'
```
Comment (eadsimone) [inline src/components/login-view/login-view.jsx]: remove duplicate type

File: src/components/login-view/login-view.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		console.log(username, password);
+		
+		// Send a request to the server for authentication, then call props.onLoggedIn(username)
+		props.onLoggedIn(username);
+	};
+	
+	return (
+		<>
```
Comment (eadsimone) [inline src/components/login-view/login-view.jsx]: it does not exist add a div instead and close div as well
```
<div>
			<Form className='login-form'>
				<h1 className='login-header'>Log in:</h1>
				<Form.Group controlId='formBasicEmail'>
					<Form.Label>Username</Form.Label>
					<Form.Control
						type='text'
						value={username}
						onChange={(e) => setUsername(e.target.value)}
						placeholder='Your username here!'
					/>
				</Form.Group>
				<Form.Group controlId='formBasicPassword'>
					<Form.Label>Password</Form.Label>
					<Form.Control
					type='password'
					value={password}
					onChange={(e) => setPassword(e.target.value)}
					placeholder='Your password here!'
					/>
				</Form.Group>
				<Button onClick={handleSubmit} variant='dark' type='submit'>Submit</Button>
			</Form>
		</div>
```

**Review by eadsimone — CHANGES_REQUESTED** [review]

**Comment by AaronRDurant** [comment]
I fixed those mistakes — thank you very much! The app now builds without errors, however nothing actually renders/shows up on the page.

**Pull request merged**